templates/single-quiz/question-type-multiple-choice.php

```diff
@@ -38,7 +38,7 @@
 			/>
 
 		<label for="<?php echo esc_attr( 'question_' . $question_data['ID'] . '-option-' . $count ); ?>">
-			<?php echo esc_html( apply_filters( 'sensei_answer_text', $option['answer'] ) ); ?>
+			<?php echo wp_kses_post( apply_filters( 'sensei_answer_text', $option['answer'] ) ); ?>
```

Comment:
Do we need "post" set of tags here?
I just think it might be too loose for this application (for example, we can add a form).
Maybe choose a more restricted set of tags?

Reply:
Yes, it makes sense.
I restricted here  af3e559101dc7be8b6946d08857e7dfc95e12ea9 to only enable the html formatting tags